ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -18,5 +18,34 @@ def swapdims(input, dim0, dim1):
     return ivy.swapaxes(input, dim0, dim1)
 
 
+swapdims.unsupported_dtypes = (
```

Comment:
I think you accidentally added this line, you're not supposed add changes to nay other function in your PR.

Reply:
removed